pkg/keyvisual/input/periodic.go

```diff
@@ -28,7 +30,8 @@ func (input *periodicInput) GetStartTime() time.Time {
 }
 
 func (input *periodicInput) Background(ctx context.Context, stat *storage.Stat) {
-	ticker := time.NewTicker(time.Minute)
+	ticker := time.NewTicker(input.interval)
+	log.Info("keyvisual load regions from periodic input", zap.Duration("interval", input.interval))
```

Comment:
How about using the dynamic config? Currently all KeyViz configurations are placed there, so that it can be updated dynamically.